core/tests/ui/e2e/order.spec.ts

```diff
@@ -0,0 +1,51 @@
+import { expect, test } from '~/tests/fixtures';
+
+function formatAmount(amount: number): string {
+  return new Intl.NumberFormat('en-US', {
+    style: 'currency',
+    currency: 'USD',
+  }).format(amount);
+}
+
+function formatDate(s: string): string {
+  return new Date(s).toLocaleDateString('en-US', {
+    year: 'numeric',
+    month: 'long',
+    day: 'numeric',
+  });
+}
+test('Orders page has empty state', async ({ page, account }) => {
+  const customer = await account.create();
+
+  await customer.login();
+
+  await page.goto('/account/orders/');
+
+  await expect(page.getByRole('heading', { name: 'Orders' })).toBeVisible();
+  await expect(page.getByText('No orders yet')).toBeVisible();
+});
+
+test('Order details are visible on Orders page', async ({ page, account, product, order }) => {
+  const customer = await account.create();
+
+  await customer.login();
+
+  const orderProduct = await product.create();
```

Comment:
Have we tried to use existing products instead of creating them programmatically?

Reply:
I didn't try that, but will do so. Do you mean something like Get all products -> filter them by stock/availability/other -> randomly get suitable product for order ? As I don't think always taking the same existing product is a good idea.

What we might improve by this is decreasing API requests amount from create, assign to channel, delete (3) to get all, and filter them (1). And bigger test data diversity. Is this the improvement you meant?

Reply:
I was just thinking about keeping it simple as we already know all our product ids, e.g. 
```ts
const orderDetails = await order.create(customer.id, 111);
```

Reply:
If we are not planning to have store provisioning for every test run any time soon, then we can go with this approach. It is indeed simpler, but makes tests dependant on specific test data. But since plenty of UI tests already depend on specific test data, we can keep it consistent then 🙂

Reply:
Yeah, I see us just using catalyst-latest for functional test suite for now. We can always reevaluate in the future if we need something like this.

Reply:
updated to use existing product. should we keep create product functionality for future use or better to remove now and we will add it in future if needed?

Reply:
done ✅